beacon-chain/core/blocks/block_operations.go

```diff
@@ -693,7 +693,7 @@ func ProcessValidatorDeposits(
 	for _, deposit := range deposits {
 		beaconState, err = ProcessDeposit(beaconState, deposit, valIndexMap, verifySignatures, true)
 		if err != nil {
-			return nil, fmt.Errorf("could not process deposit index %d: %v", deposit.Index, err)
+			return nil, fmt.Errorf("could not process deposit from %#x: %v", deposit.Data.Pubkey, err)
```

Comment:
Would be good to `Trunc` the public key here (similar to what we did across validator clients), otherwise it's too long to log